build-logic/packaging/src/main/kotlin/gradlebuild.distributions.gradle.kts

```diff
@@ -262,8 +263,12 @@ fun configureDistribution(name: String, distributionSpec: CopySpec, buildDistLif
     val zipRootFolder = if (normalized) {
         moduleIdentity.version.map { "gradle-${it.baseVersion.version}" }
     } else {
-        moduleIdentity.version.map { "gradle-${it.version}" }
+        moduleIdentity.version.map { "gradle-${it.version}" }.map {
```

Comment:
🤔 Are you sure such a distribution is compatible with the wrapper when downloading it from services.gradle.org? Doesn't that expect a folder with the same name as the version inside?